src/main/scala/org/allenai/deep_qa/experiments/SciQDatasetExperiments.scala

```diff
@@ -0,0 +1,302 @@
+      ) ~
+      ("optimizer" ->
+        ("type" -> "adam") ~
+        ("clipnorm" -> 10.0) ~
+        ("lr" -> 0.0005)
+      ) ~
+      ("embedding_dropout" -> 0.0) ~
+      ("patience" -> 0) ~
+      ("num_epochs" -> 5)
+    )
+  }
+
+
+  def main(args: Array[String]) {
+    runASReaderOmnibusEightExperiment
+    runASReaderOmnibusFourExperiment
+    runGAReaderOmnibusEightExperiment
+    runGAReaderOmnibusFourExperiment
+  }
+
+  def runASReaderOmnibusFourExperiment() {
+    val asReaderOmnibusFourDefault = omnibusGradeFourExperiment(
+      "ASReader omnibus four",
+      attentionSumReader,
+      omnibusFourTrainReadingComprehensionDataset
+    )
+
+    val asReaderOmnibusFourWithSciQDataset = omnibusGradeFourExperiment(
+      "ASReader omnibus four plus SciQ Dataset",
+      attentionSumReader,
+      combinedSciQTrainOmnibusFourTrainDataset
+    )
+
+    val models = Seq(asReaderOmnibusFourDefault, asReaderOmnibusFourWithSciQDataset)
+    new Evaluator(Some("ASReader_omnibus_four_plus_sciq_dataset"), models, fileUtil).runPipeline()
+  }
+
+  def runASReaderOmnibusEightExperiment() {
+    val asReaderOmnibusEightDefault = omnibusGradeEightExperiment(
+      "ASReader omnibus eight",
+      attentionSumReader,
+      omnibusEightTrainReadingComprehensionDataset
+    )
+
+    val asReaderOmnibusEightWithSciQDataset = omnibusGradeEightExperiment(
+      "ASReader omnibus eight plus SciQ Dataset",
+      attentionSumReader,
+      combinedSciQTrainOmnibusEightTrainDataset
+    )
+
+    val models = Seq(asReaderOmnibusEightDefault, asReaderOmnibusEightWithSciQDataset)
+    new Evaluator(Some("ASReader_omnibus_eight_plus_sciq_dataset"), models, fileUtil).runPipeline()
+  }
+
+    def runGAReaderOmnibusFourExperiment() {
+    val gaReaderOmnibusFourDefault = omnibusGradeFourExperiment(
+      "GAReader omnibus four",
+      gatedAttentionReader,
+      omnibusFourTrainReadingComprehensionDataset
+    )
+
+    val gaReaderOmnibusFourWithSciQDataset = omnibusGradeFourExperiment(
+      "GAReader omnibus four plus SciQ Dataset",
+      gatedAttentionReader,
+      combinedSciQTrainOmnibusFourTrainDataset
+    )
+
+    val models = Seq(gaReaderOmnibusFourDefault, gaReaderOmnibusFourWithSciQDataset)
+    new Evaluator(Some("GAReader_omnibus_four_plus_sciq_dataset"), models, fileUtil).runPipeline()
+  }
+
+  def runGAReaderOmnibusEightExperiment() {
+    val gaReaderOmnibusEightDefault = omnibusGradeEightExperiment(
+      "GAReader omnibus eight",
+      gatedAttentionReader,
+      omnibusEightTrainReadingComprehensionDataset
+    )
+
+    val gaReaderOmnibusEightWithSciQDataset = omnibusGradeEightExperiment(
+      "GAReader omnibus eight plus SciQ Dataset",
+      gatedAttentionReader,
+      combinedSciQTrainOmnibusEightTrainDataset
+    )
+
+    val models = Seq(gaReaderOmnibusEightDefault, gaReaderOmnibusEightWithSciQDataset)
+    new Evaluator(Some("GAReader_omnibus_eight_plus_sciq_dataset"), models, fileUtil).runPipeline()
+  }
+
```

Comment:
Minus some nitpicks about where code should go, this experiment was set up quite nicely. Now that you've done this, do you have any feedback on the experiment creation process?  Good?  Bad?  What can be improved?